src/js/utils/browser.ts

```diff
@@ -78,26 +78,29 @@ export function flashVersion(): number {
     }
 
     const plugins = navigator.plugins;
-    let flash;
 
     if (plugins) {
-        flash = plugins.namedItem('Shockwave Flash');
-        if (flash && flash.description) {
-            return parseFloat(flash.description.replace(/\D+(\d+\.?\d*).*/, '$1'));
+        const flashPlugin = plugins.namedItem('Shockwave Flash');
+        if (flashPlugin && flashPlugin.description) {
+            return parseFloat(flashPlugin.description.replace(/\D+(\d+\.?\d*).*/, '$1'));
         }
     }
 
     if (typeof window.ActiveXObject !== 'undefined') {
         try {
-            flash = new window.ActiveXObject('ShockwaveFlash.ShockwaveFlash');
-            if (flash) {
-                return parseFloat(flash.GetVariable('$version').split(' ')[1].replace(/\s*,\s*/, '.'));
+            const flashObj = new window.ActiveXObject('ShockwaveFlash.ShockwaveFlash');
+            if (flashObj) {
+                return parseFloat((flashObj as FlashObject).GetVariable('$version').split(' ')[1].replace(/\s*,\s*/, '.'));
             }
         } catch (e) {
             return 0;
         }
 
-        return flash;
+        return 0;
```

Comment:
Correct me if I'm wrong, but it seems like `flash` at this point would either be `undefined` or the `flashPlugin` if it didn't have a description. Those should probably return as `0`

Reply:
`flash` could have been set to `plugins.namedItem('Shockwave Flash')` and not have a `description` property so we return that value here.